cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/SingleBrokerStats.java

```diff
@@ -45,6 +54,15 @@ BasicStats basicStats() {
     return _basicStats;
   }
 
+  Map<String, Double> utilByDisk() {
+    return _utilByDisk;
+  }
+
+  double pctForDisk(String logdir) {
```

Comment:
Nit: `pctForDisk` -> `pctFor` (i.e. it seems to be the percentage for the given `logdir`, which might contain multiple disks in case of `RAID`?)